esp/templates/program/modules/studentclassregmodule/catalog.html

```diff
@@ -153,6 +167,19 @@ <h2 style="text-align: center;">
 hideClassesOutOfGradeRange(student_grade);
 </script>
 
+<script type="text/javascript">
+$j("#grade_filter").change(function(){
```

Comment:
The script right above this one seems to be doing approximately the same thing (except using the student's current grade).  Why do we need a new one? Is it just to allow looking at a different grade?  Can we remove the old one?  Can we make sure it still defaults to showing the current user's grade, if they are logged in?

Reply:
Heh, I think it's a global thanks to the wonders of JavaScript.  From searching the codebase it looks like it's set here: https://github.com/learning-unlimited/ESP-Website/blob/9d40eedecac28ad895db5518f5b83afb68813a99/esp/templates/program/modules/studentclassregmodule/catalog.html#L21

Reply:
Great, thanks for cleaning this up!  Happy to see a big pile of code go away :-) .